supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:managed-nodes[managed-nodes]
+
+[discrete]
+[[kerberos-authentication-indicators]]
+==== image:images/yes.png[yes] Kerberos authentication indicators (noun)
+*Description*: Authentication indicators are attached to Kerberos tickets and represent the initial authentication method used to acquire a ticket:
+
+* `otp` for two-factor authentication (password + One-Time Password)
+* `radius` for Remote Authentication Dial-In User Service (RADIUS) authentication (commonly for 802.1x authentication)
+* `pkinit` for Public Key Cryptography for Initial Authentication in Kerberos (PKINIT), smart card, or certificate authentication
+* `hardened` for passwords hardened against brute-force attempts
+
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html/configuring_and_managing_identity_management/managing-kerberos-ticket-policies_configuring-and-managing-idm#kerberos-authentication-indicators_managing-kerberos-ticket-policies[Kerberos authentication indicators].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[kerberos-keytab]]
+==== image:images/yes.png[yes] Kerberos keytab (noun)
+*Description*: While a password is the default authentication method for a user, keytabs are the default authentication method for hosts and services. A Kerberos keytab is a file that contains a list  of Kerberos principals and their associated encryption keys, so a service can retrieve its own Kerberos key and verify a user’s identity.
+
+For example, every IdM client has an `/etc/krb5.keytab` file that stores information about the `host` principal, which represents the client machine in the Kerberos realm.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos-principal[kerberos-principal]
+
+[discrete]
+[[kerberos-principal]]
+==== image:images/yes.png[yes] Kerberos principal (noun)
+*Description*: Unique Kerberos principals identify each user, service, and host in a Kerberos realm:
+
+[cols="15,40,45",options="header"]
+|===
+|Entity
+|Naming convention
+|Example
+
+|Users
+a|`identifier@REALM`
+a|`admin@EXAMPLE.COM`
+
+|Services
+a|`service/fully-qualified-hostname@REALM`
+a|`http/server.example.com@EXAMPLE.COM`
+
+|Hosts
+a|`host/fully-qualified-hostname@REALM`
+a|`host/client.example.com@EXAMPLE.COM`
+|===
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos-realm[kerberos-realm]
+
+[discrete]
+[[kerberos-protocol]]
+==== image:images/yes.png[yes] Kerberos protocol (noun)
+*Description*: Kerberos is a network authentication protocol that provides strong authentication for client and server applications by using secret-key cryptography. IdM and Active Directory use Kerberos for authenticating users, hosts and services.
```

Comment:
?

> authenticating users, hosts, and services."

Reply:
What do you think is wrong with that, Darragh?

Reply:
HI @Sarka-Jana . Usually we add the a comma before the last item on a list, unless `hosts and services` is a compound noun?

Reply:
Thank you, Darragh, I see it now. Need new glasses :-D